web/yo/app/scripts/controllers/gene.js

```diff
@@ -105,6 +105,19 @@ angular.module('oncokbApp')
                     console.log('Unable to add comment.');
                 }
             };
+            $scope.vusUpdate = function() {
+                if ($scope.status.isDesiredGene) {
+                    var vus = $scope.realtimeDocument.getModel().getRoot().get('vus');
+                    var vusData = stringUtils.getVUSFullData(vus, true);
+                    DatabaseConnector.updateVUS($scope.gene.name, JSON.stringify(vusData), function(result) {
+                        console.log('success saving vus to database');
+                    }, function(error) {
+                        console.log('error happened when saving VUS to DB', error);
+                        sendEmail('jiaojiaowanghere@gmail.com', 'Error happened when saving VUS for ' + $scope.gene.name.getText(), error);
```

Comment:
If you want to send an email to you, you probably also want to know which VUS and what had been done to that VUS, right? (Like V600A has been updated by WHOM on **/**/****, the old value was ********) Otherwise, you need to ask the curator or admins.